apps/amiapp_flutter/lib/src/utils/extensions.dart

```diff
@@ -99,3 +100,16 @@ extension AmiAppDoubleExtensions on double {
     return toStringAsFixed(0);
   }
 }
+
+extension LocationExtensions on GoRouter {
+  // Get location of current route
+  // This is a workaround to get the current location as location property
```

Comment:
Since this is labeled as a workaround, it makes me wonder if this is a short-term solution? To make our sample app an up-to-date reference for customers, should we replace this with recommended best practice code in the future?

Reply:
Agreed. I'm not expert with `GoRouter` so not fully sure about best practices for it at the moment. But yes, we can put at a ticket to simplify and improve this for customers. Sample apps project could be the best time to handle this? 🤔